lib/factory_girl/registry.rb

```diff
@@ -13,6 +13,10 @@ def clear
       @items.clear
     end
 
+    def remove(name)
+      @items.delete(name.to_sym)
```

Comment:
Thoughts on whether this should raise if you attempt to delete something that doesn't exist, similar to `find`?